src/modules/auth/index.js

```diff
@@ -5,20 +5,22 @@ import reducer from './reducer';
 import { types } from './actions'
 import { SeaCatAuthApi, GoogleOAuth2Api } from './api';
 
-export default class AuthModule extends Module {
 
+export default class AuthModule extends Module {
 
 	constructor(app, name){
 		super(app, "AuthModule");
 
 		this.OAuthToken = JSON.parse(sessionStorage.getItem('SeaCatOAuth2Token'));
 		this.UserInfo = null;
-		this.Api = new SeaCatAuthApi(app.Config);
+		this.SeaCatAuthApi = new SeaCatAuthApi(app.Config);
```

Comment:
Nope, it has to still be `Api` since other providers such as Google OAuth could be used eventually.